app/views/candidates/show.html.erb

```diff
@@ -18,20 +15,20 @@
       <div class="card text-center p-2 candidate-info">
         <div class="card-body">
           <h3 class="card-text"><strong>Email: </strong><%= @candidate.email %></h3>
-          <h3 class="card-text"><strong>Phone Number: </strong><%= @candidate.phone %></h3>
+          <h3 class="card-text"><strong>Phone Number: </strong><%= @candidate.phone_number %></h3>
           <h3 class="card-text"><strong>University: </strong><%= @candidate.university %></h3>
           <h3 class="card-text"><strong>GPA: </strong><%= @candidate.gpa %></h3>
-          <h3 class="card-text"><strong>Bio: </strong><%= @candidate.bio %></h3>
           <h3 class="card-text"><strong>Status: </strong><%= @candidate.status %></h3>
+          <h3 class="card-text text-start"><strong>Bio: </strong><%= @candidate.bio %></h3>
         </div>
       </div>
 
-      <div class="container add-border p-3">
+      <div class="container add-border p-5">
         <div class="row">
           <div class="col-3">
-            <% if @candidate.picture.attached? %>
+            <% if @candidate.cv.attached? %>
               <% puts "==================== CV IS ATTACHED ====================" %>
```

Comment:
we dont use puts in html file you can use p tag for this

Reply:
Just a message to print on console for testing, have removed it